inference-engine/ie_bridges/python/install/setup.py.in

```diff
@@ -0,0 +1,249 @@
+import os.path
```

Comment:
run pylint

Reply:
************* Module setup
setup.py:37:0: C0301: Line too long (101/100) (line-too-long)
setup.py:38:0: C0301: Line too long (104/100) (line-too-long)
setup.py:95:0: C0115: Missing class docstring (missing-class-docstring)
setup.py:102:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
setup.py:95:0: R0903: Too few public methods (0/2) (too-few-public-methods)
setup.py:222:18: W0612: Unused variable 'directories' (unused-variable)

------------------------------------------------------------------
Your code has been rated at 9.62/10 (previous run: 7.73/10, +1.88)

Reply:
FYI, you should be able to eliminate the second one by renaming `directories` to `_`. Although my advice would be to get rid of `os.walk` entirely and use `local_base_dir.glob` instead.

Reply:
In lines [215-221 of setup.py](https://github.com/openvinotoolkit/openvino/pull/3582/files#diff-192de7534c254340555259c731cdcc4ca429bbde7bacb1b3050e88d93d190e38R215-R221) you are actually not using directories anywhere. Try using `_directories` or simple put underscore there so pylint will process it as a dummy variable. ([https://docs.pylint.org/en/1.6.0/features.html#id17](https://docs.pylint.org/en/1.6.0/features.html#id17) under dummy-variables-rgx)

Reply:
For **R0903: Too few public methods** there is a workaround by setting `min-public-methods=0` in .pylintrc file as I remember.